src/routes/index.js

```diff
@@ -1,5 +1,8 @@
 const router = require("express").Router();
 
 router.use("/api", require("./api"));
+router.get('/', (req, res) => {
+    res.status(200).json({status:200, message:'welcome'});
```

Comment:
You don't need the 'status:200' entry since your response status is already 200

Reply:
> You don't need the 'status:200' entry since your response status is already 200

thank you @karamuka am going to modify it